pkg/vault/vault_interface_test.go

```diff
@@ -21,7 +22,7 @@ func Log(t *testing.T, label string, err error) {
 
 func TestCredentialManagerInterface(t *testing.T) {
 	var err error
-	os.Setenv("RUN_WITHOUT_VAULT", "0")
+	os.Setenv("RUN_WITHOUT_VAULT", "1")
```

Comment:
We want to test real vault connection, not the dummy interface.

Reply:
Yes. Is there a real vault instance we can run against? E.g. in the integration test one could run against vault and in the unit test against the dummy?

Reply:
Both helm and vault packages are tested in integration tests only, not in unit tests, if I am not mistaken.

Reply:
They are currently not tested at all. And for the integration tests they need an exposed port that currently is not exposed in the integration tests.